src/content/roadmap/account-abstraction/index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Account abstraction
+description: An overview of Ethereum's plans to make user accounts simpler and safer
+lang: en
+template: staking
+emoji: ":money_with_wings:"
+image: ../../../assets/staking/leslie-pool.png
+alt: Leslie the rhino swimming in the pool.
+sidebarDepth: 2
+summaryPoints:
+  - Understand why account abstraction will make Ethereum easier to use
+  - Learn about upcoming changes to how users will interact with Ethereum
+  - Read why account abstraction will help users to secure their assets
+---
+
+Today, users interact with Ethereum using **externally owned accounts (EOAs)**. These are the only entities that can initiate a transaction or execute functions in smart contracts, and they pay gas fees in ETH whenever those transactions change Ethereum's state. Account abstraction aims to relax this constraint to allow smart contract wallets to initiate transactions too. This allows users to flexibly program more security and better user experiences into their accounts.
+
+Account abstraction is an upgrade that makes smart contract wallets first-class citizens, natively supported on Ethereum. Those smart contract wallets unlock many benefits for the user, including:
+
+- no more seed phrases
+- define your own flexible security rules
+- recover your account if you lose the keys
+- share your account security across trusted devices or individuals
+- pay someone else's gas, or have someone else pay yours
+- easily batch transactions together (e.g. approve and execute a swap in one go)
+- more opportunities for dapps and wallet developers to innovate on user experiences
+- upgrade the cryptography securing an account when quantum computers arrive
+
+These benefits are not natively supported today because Ethereum accounts are simple public-private key pairs. The security model is simple: if you have the private key you can do _anything_ within the rules of the Ethereum Virtual Machine (EVM); if you do not have the private key you can do _nothing_. This means lost keys are completely unrecoverable and stolen keys give thieves instant access to all the funds in an account. The logic that can be embedded in these accounts is only the validation that is native to the EVM and cannot be customized. Smart contract wallets are the solution to these problems, but today they are difficult to program because in the end any logic they implement has to resolve to a sequence of EOA transactions before they can be processed by Ethereum. Account abstraction enables smart contracts to initiate transactions themselves, so that any logic that the user wishes to implement can be coded into the smart contract wallet itself and executed on Ethereum.
```

Comment:
```suggestion
These benefits are not natively supported today because Ethereum accounts (EOA's) are simply public-private key pairs. They work like this: 

- if you have the private key you can do _anything_ within the rules of the Ethereum Virtual Machine (EVM)
- if you do not have the private key you can do _nothing_. 

If you lose your keys they can't be recovered, and stolen keys give thieves instant access to all the funds in an account. 

Smart contract wallets are the solution to these problems, but today they are difficult to program because in the end any logic they implement has to be translated into a set of EOA transactions before they can be processed by Ethereum. Account abstraction enables smart contracts to initiate transactions themselves, so that any logic that the user wishes to implement can be coded into the smart contract wallet itself and executed on Ethereum.
```